README.md

```diff
@@ -0,0 +1,24 @@
+# GOV.UK Repository Mirrorer
+
+Ensures all repositories tagged with "govuk" in the alphagov organisation in Github.com are mirrored to Gitlab.com.
+
+We use Gitlab.com so we can still deploy in the event that Github.com is down. We also use the private repositories at Gitlab in order to fix any security vulnerabilities without disclosing them.
```

Comment:
Perhaps a link to https://docs.publishing.service.gov.uk/manual/github-unavailable.html or https://docs.publishing.service.gov.uk/manual/deploy-fixes-for-a-security-vulnerability.html would be good here.